.github/workflows/ci.yml

```
@@ -9,24 +9,43 @@ on:
       - '**'
 
 jobs:
-  lint:
+  lint-main:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: services/main-service
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
           node-version: '22'
-      - run: npm ci
+      - run: npm install
```

```suggestion
      - run: npm ci
```
Краще інстальовувати ті ж версії пакетів, які ти вже протестував під час розробки